client/src/components/DashBoard/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+
+class DashBoard extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    componentDidMount() {
+        // Do the API call here
+    }
+
+    getDisplayLocation() {
+        
```

Comment:
Why do we have this function if we are not using it?

Reply:
I know it is a bit redundant for now but will be used later. I just kept it in there to have a structure moving forward.